specification/trafficmanager/resource-manager/Microsoft.Network/stable/2018-08-01/trafficmanager.json

```diff
@@ -967,6 +967,16 @@
           "format": "int64",
           "description": "The minimum number of endpoints that must be available in the child profile in order for the parent profile to be considered available. Only applicable to endpoint of type 'NestedEndpoints'."
         },
+        "minChildEndpointsIPv4": {
```

Comment:
@allencal Can you add examples using these newly added properties?

Reply:
Done! Added Profile-PUT-WithNestedEndpoints.json